plugin-packs/postcss-preset-env/test/layers-basic.preserve.true.expect.css

```diff
@@ -785,6 +784,8 @@ h1.test-custom-selectors:not(#\#):not(#\#):not(#\#):not(#\#):not(#\#):not(#\#):n
 }
 
 .unset:not(#\#):not(#\#):not(#\#):not(#\#):not(#\#):not(#\#):not(#\#):not(#\#):not(#\#):not(#\#):not(#\#):not(#\#):not(#\#):not(#\#) {
+	clip: auto;
+	clip: initial;
```

Comment:
No idea why this changed in this way.
But it looks more correct.

I don't really understand where `clip: initial` is suddenly coming from. It is the correct fallback for `unset`, but I didn't make any change to add this. It should have always been there.

Maybe a bug in `postcss-initial` where it removes nodes and somehow I don't have the same bug?